part-2-server/server.js

```diff
@@ -0,0 +1,43 @@
+var http = require("http");
+var fs = require("fs");
+
+var server = http.createServer(function(req, res) {
+    if (req.url.match(/\/poem-/)) {
+        var target = req.url.split("");
+        target.shift();
+        target = target.join("");
+        target = target + ".txt"
+        fs.readFile(target, function(err, data) {
+            res.write(data.toString());
+            res.end();
+        })
+    }
+
```

Comment:
This is a little hard to follow...would be helpful to either save more descriptive variables, or separate this logic out as a separate function